File: Examples/CSharp/AsposePDF/Forms/AddTooltipToField.cs
//////////////////////////////////////////////////////////////////////////
// Copyright 2001-2013 Aspose Pty Ltd. All Rights Reserved.
//
// This file is part of Aspose.Pdf. The source code in this file
// is only intended as a supplement to the documentation, and is provided
// "as is", without warranty of any kind, either expressed or implied.
//////////////////////////////////////////////////////////////////////////
using System.IO;

using Aspose.Pdf;
using Aspose.Pdf.InteractiveFeatures.Forms;

namespace CSharp.AsposePdf.Forms
{
    public class AddTooltipToField
    {
        public static void Run()
        {
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_AsposePdf_Forms();

            // Load source PDF form
            Document doc = new Document(dataDir + "AddTooltipToField.pdf");

            // Set the tooltip for textfield
            (doc.Form["textbox1"] as Field).AlternateName = "Text box tool tip";

            // Save the updated document
            doc.Save(dataDir + "AddTooltipToField_out.pdf");

            
        }
    }
}

File: Examples/VisualBasic/AsposePDF/Forms/AddTooltipToField.vb
Imports System.IO
Imports System
Imports Microsoft.VisualBasic
Imports Aspose.Pdf.Forms
Imports Aspose.Pdf

Namespace AsposePDF.Forms
    Public Class AddTooltipToField
        Public Shared Sub Run()
            ' ExStart:AddTooltipToField
            ' The path to the documents directory.
            Dim dataDir As String = RunExamples.GetDataDir_AsposePdf_Forms()

            ' Load source PDF form
            Dim doc As New Document(dataDir & Convert.ToString("AddTooltipToField.pdf"))

            ' Set the tooltip for textfield
            TryCast(doc.Form("textbox1"), Field).AlternateName = "Text box tool tip"

            dataDir = dataDir & Convert.ToString("AddTooltipToField_out_.pdf")
            ' Save the updated document
            doc.Save(dataDir)
            ' ExEnd:AddTooltipToField   
            Console.WriteLine(Convert.ToString(vbLf & "Tooltip added successfully." & vbLf & "File saved at ") & dataDir)
        End Sub
    End Class
End Namespace
